platform/vs/docker-sonic-vs/platform.json

```diff
@@ -3,194 +3,350 @@
         "Ethernet0": {
             "index": "0,0,0,0",
             "lanes": "25,26,27,28",
-            "alias_at_lanes": "fortyGigE0/0,fortyGigE0/1,fortyGigE0/2,fortyGigE0/3",
-            "breakout_modes": "1x100G[40G],2x50G,4x25G[10G],2x25G(2)+1x50G(2),1x50G(2)+2x25G(2)"
+            "breakout_modes": {
+                "1x100G[40G]": ["fortyGigE0/0"],
+                "2x50G": ["fortyGigE0/0", "fortyGigE0/1"],
```

Comment:
should be fortygige0/0 and fortygige0/2

Reply:
What is the significant of this values especially for testing?   If we can make it work for vs tests, it should be fine?

I know different platforms/vendors have different convention,  e,g, some main vendor uses 0/1 and 0/2 for 2x50G instead of 0/1 and 0/3

Reply:
Modified as requested.